lms/static/sass/shared/_footer-edx.scss

```diff
@@ -19,20 +19,72 @@ footer#footer-edx-v3 {
   padding: $baseline*0.75 0;
   border-top: 1px solid $courseware-button-border-color;
 
+  .column-1 {
+    flex: 0 0 100%;
+    max-width: 100%;
+  }
+
+  .column-styles {
+    position: relative;
+    width: 100%;
+    min-height: 1px;
+    padding-right: 15px;
+    padding-left: 15px;
+  }
+
+  .column-2 {
+    flex: 0 0 100%;
+    max-width: 100%;
+  }
+
+  .column-3 {
+    flex: 0 0 50%;
+    max-width: 50%;
+  }
```

Comment:
This is the only class that doesn't have;
```
flex: 0 0 50%;
max-width: 100%;
```
I would suggest defaulting to those settings and then updating ```.column-3``` only (probably using ```&.column-3``` nested within ```.column-styles```.

Reply:
Can you explain please it a little ?  Thanks